recipe/meta.yaml

```diff
@@ -1,33 +1,35 @@
 {% set name = "lsst-efd-client" %}
-{% set version = "0.12.0" %}
+{% set version = "0.13.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lsst-efd-client-{{ version }}.tar.gz
-  sha256: e304340eae2ca5fc390646278f2bb31679e799cabf0a3d2d0e778a8cdc36b141
+  sha256: ffb7c0bbec02e0b36e488dd4e0c695b0f3fe24c77106e688f8d2cff92c8e7af7
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
+    - setuptools
+    - setuptools-scm
     - pip ==19.2.3
-    - bump2version ==0.5.11
     - wheel ==0.33.6
+    - bump2version ==0.5.11
     - watchdog ==0.9.0
     - flake8 ==3.7.8
     - coverage ==4.5.4
     - twine
     - pytest-runner
   run:
-    - python >=3.5
-    - numpy >=1.16.5
+    - python >=3.7
+    - numpy <1.26
```

Comment:
So this is to make up for aioinflux's lack of pinning and incompatibility with >=1.26 due to use of `np.float` instead of `np.float64` or `float`, correct?  If so, I think this is the best that can be done until we get rid of aioinflux.

Reply:
Yes, exactly. You could also modify the aoinflux recipe (which I think you're the maintainer for) but I didn't find that out until later. There's no reason at all that the lsst-efd-client code itself needs numpy< 1.26.

Reply:
Hopefully this can be dealt with by @kkeijuro once he's working on this stuff again.

Reply:
I think once there is a new tagged version of lsst-efd-client that actually uses the pyproject.toml file and build setup that is currently on main, that most of this recipe would go away and be simplified back to what I originally tried.